### DISCUSSION TITLE
Show your Serpent

### DISCUSSION THREAD

**Discussion by NomadicCore** [AerynOS/discussions#3] [Show and tell]
As the name implies, just a fun thread to show your Serpent install and setup's.  I will go first:

![screenshot-2024-08-21-22-33-51](https://github.com/user-attachments/assets/2d7fc134-c2f2-4df7-84bc-4f92aa608b44)
![screenshot-2024-08-21-22-35-07](https://github.com/user-attachments/assets/268a0d3f-45f8-4998-ad11-6b9b7fb47b00)

**Comment by ikeycode** [comment]
![Screenshot from 2024-08-21 22-46-23](https://github.com/user-attachments/assets/57b2ebe6-4110-43c5-8106-85925e62492f)

**Comment by palasso** [comment]
![Screenshot_Serpent_2024-08-22_23:52:38](https://github.com/user-attachments/assets/2889a00f-341a-49ea-bd2f-8fb7b367b094)

**Reply by NomadicCore** [reply]
I like that panel configuration with the apps in the centre and time over on the right next to power and no app launcher on the left. I'm pretty much exclusively using the super key anyway so have copied that! Thanks for sharing!

**Comment by jplatte** [comment]
![Screenshot from 2024-08-23 00-20-47](https://github.com/user-attachments/assets/fd6e15fc-fe55-4f56-a875-b3cbc6f21346)

**Comment by jhmorenof** [comment]
This works very well for me. I just don't have a printer... and I need one. I will be patient. Thanks great team of developers!
![screenshot-2024-09-29-15-24-19](https://github.com/user-attachments/assets/d5188cb8-6e65-40cb-a908-4ead452dc002)

**Comment by ikeycode** [comment]
Heathen time - GNOME + tiling xD

![Screenshot From 2024-11-22 14-32-42](https://github.com/user-attachments/assets/059a153a-ebfb-478e-8292-0edcbc14b86c)

**Reply by ermo** [reply]
I mean, GNOME is essentially a perfect fit for a tiling WM.

**Reply by NomadicCore** [reply]
Can't wait to try tiling when my new monitor comes in! Will be perfect candidate for it. 

I wonder if Cosmic or Gnone tiling has options for more advanced presets or if this is something that is achievable via further apps in the future when the repo expands?

**Comment by ghosts-haunted** [comment]
All the things...
Click play & move mouse away

https://github.com/user-attachments/assets/6e0b833e-5167-4a77-8938-94b89590fe0d

![Screenshot From 2025-01-16 18-06-02](https://github.com/user-attachments/assets/c8479d1a-0f45-464c-9160-72b2cde780e3)

**Comment by Ghodawalaaman** [comment]
[packaging-proof.webm](https://github.com/user-attachments/assets/d00ed04c-67b7-4153-a9b1-c30d54a836cc)

Running sway in Serpent OS :D
